src/private/MDMPerformerGroup.h

```diff
@@ -37,7 +37,7 @@
 // nil by default. Useful for view duplication.
 @property(nonatomic, nullable) id schedulerTarget;
 
-- (void)addPlan:(nonnull id<MDMPlan>)plan trace:(nonnull MDMTrace *)trace;
+- (void)addPlan:(nonnull id<MDMPlan>)plan trace:(nonnull MDMTrace *)trace log:(nullable MDMTransactionLog *)log;
```

Comment:
Hmmm. I guess I was mostly using `- (void)addPlan:(nonnull id<MDMPlan>)plan trace:(nonnull MDMTrace *)trace log:(nullable MDMTransactionLog *)log` for _everything_ and then twisting and turning within that method with if checks for each of the different variants. I'll have a look into breaking this out.